mds_asssignment_1/guestbook/views.py

```diff
@@ -0,0 +1,29 @@
+from django.shortcuts import render
+
+from .forms import EntryForm
+from .models import Entry
+
+
+def home(request):
+    entry_list = Entry.objects.all().filter(visible=True).order_by('date').reverse()
+    if request.method == 'POST':
+        entry_form = EntryForm(request.POST)
+        if entry_form.is_valid():
+            entry = Entry()
```

Comment:
You could also do this. You wouldn't need to call `save()` on `entry` at the end.
```python
entry = Entry.objects.create(
    fname=entry_form.cleaned_data['fname'],
    lname=entry_form.cleaned_data['lname'],
    email=entry_form.cleaned_data['email'],
    comment=entry_form.cleaned_data['comment']
)
```